README.md

```diff
@@ -81,34 +88,51 @@ We recommend using [VSCode](https://code.visualstudio.com/) with the following e
 
 ## Deployments
 
-Frontend: After running `npm run build`, push to your `main` branch to deploy the app. Heroku uses the contents of the `dist` folder for deployment.
-Backend: Pushing to the `main` branch will automatically trigger a new Heroku build.
+Frontend: After running `npm run build`, push to your `main` branch to deploy the app.
+Backend: `Amplify push` will automatically trigger a new Amplify build.
 
-### View Heroku logs
+### View Amplify logs
 
-```
-$ heroku logs --tail
+- Install the AWS Amplify CLI:
+
+```bash
+$ npm install -g @aws-amplify/cli
 ```
 
-### Automated Tests
+- Configure the Amplify CLI:
 
-Testing is crucial for ensuring the reliability and stability of the application. In this project, automated tests have been set up to make the validation process more efficient. Make sure all dependencies are installed and the application's environment is set up correctly for the tests to run without issues.
+```bash
+$ amplify configure
+```
+
+- Check Amplify status:
 
+```bash
+$ amplify status
 ```
-##TOOLS USED##
 
-Jest: A delightful JavaScript Testing Framework with a focus on simplicity.
-Supertest: Provides a high-level abstraction for testing HTTP, while still allowing you to drop down to the lower-level API provided by superagent
+- View Amplify logs: To view the logs for a specific Amplify function or resource, you can use the amplify console command. For instance, if you want to view logs for a Lambda function:
 
+```bash
+$ amplify console function
 ```
 
-##HOW TESTS WORK##
+## Automated Tests
 
-Jest acts as the test runner, providing an environment to execute our tests.
-With Supertest, we can easily make HTTP requests to our API and assert the responses, making it suitable for testing RESTful services.
+Testing is crucial for ensuring the reliability and stability of the application. In this project, automated tests have been set up to make the validation process more efficient. Ensure all dependencies are installed and the application's environment is set up correctly for the tests to run without issues.
 
-```
-##RUNNING THE TESTS##
+### Tools Used for API Testing
+
+- **Jest**: A delightful JavaScript Testing Framework with a focus on simplicity.
+- **Supertest**: Provides a high-level abstraction for testing HTTP, while still allowing you to drop down to the lower-level API provided by superagent.
+- **vitest**: Vitest is a unit-test framework tailored for Vite, utilizing Vite's dev server for file transformations and offering rapid testing with features like Hot Module Reload (HMR). With a Jest-compatible API, it emphasizes performance and developer experience, aiming to be the preferred test runner for Vite projects.
```

Comment:
vitest is only for ui testing. Let's separate theise out. Also make sure to specify the commands for how to test